package.json

```diff
@@ -24,18 +24,18 @@
   },
   "bundleDependencies": [
     "workshopper-adventure",
-    "concat-stream",
```

Comment:
@ccarruitero should I add the new dependencies here? 🤔

Reply:
I'm not sure that `bundleDependencies` are used for any process; but add the new dependency there, just in case.